controllers/datafoundationclient.go

```diff
@@ -144,6 +158,20 @@ func (r *dataFoundationClientReconciler) reconcilePhases() (ctrl.Result, error)
 	}
 }
 
+func (r *dataFoundationClientReconciler) reconcileWatchers() error {
```

Comment:
Why is this function named reconcile? It does not do any kind of reconciliation. Also, why do we need the function at all? this code should be embedded at the start of reconcilePhases.